retis/src/core/probe/kernel/kernel.rs

```diff
@@ -158,18 +151,18 @@ impl RawEventSectionFactory for KernelEventFactory {
 pub(crate) mod benchmark {
     use anyhow::Result;
 
-    use super::RawKernelEvent;
     use crate::{
         benchmark::helpers::*,
+        bindings::common_uapi::kernel_event,
         core::{events::FactoryId, kernel::Symbol},
     };
 
-    impl RawSectionBuilder for RawKernelEvent {
+    impl RawSectionBuilder for kernel_event {
         fn build_raw(out: &mut Vec<u8>) -> Result<()> {
-            let data = RawKernelEvent {
+            let data = Self {
                 symbol: Symbol::from_name("openvswitch:ovs_do_execute_action")?.addr()?,
-                r#type: 2, // Raw tracepoint.
-                stack_id: u64::MAX,
+                type_: 2, // Raw tracepoint.
+                stack_id: u64::MAX as i64,
```

Comment:
I think we should just use `-1` here as this is the "no stack available" value. This doesn't change runtime behavior as the stack handling is gated for tests.